pykechain/models/property.py

```diff
@@ -40,6 +42,9 @@ class Property(Base):
     :type is_invalid: bool
     """
 
+    use_bulk_update = False
+    _update_package = list()
```

Comment:
Why don't we only implement this in `Property2` as I would love to deprecate `Property` rather sooner than later.